conf/app.borderMeansOfTransport.routes

```diff
@@ -0,0 +1,4 @@
+GET         /border-means-of-transport/add-conveyance-reference-number/:departureId           controllers.borderMeansOfTransport.AddConveyanceReferenceYesNoController.onPageLoad(departureId: String, mode: Mode = NormalMode)
+POST        /border-means-of-transport/add-conveyance-reference-number/:departureId           controllers.borderMeansOfTransport.AddConveyanceReferenceYesNoController.onSubmit(departureId: String, mode: Mode = NormalMode)
+GET         /border-means-of-transport/change-conveyance-reference-number/:departureId        controllers.borderMeansOfTransport.AddConveyanceReferenceYesNoController.onPageLoad(departureId: String, mode: Mode = CheckMode)
+POST        /border-means-of-transport/change-conveyance-reference-number/:departureId        controllers.borderMeansOfTransport.AddConveyanceReferenceYesNoController.onSubmit(departureId: String, mode: Mode = CheckMode)
```

Comment:
@karahanozturk Re lines 3 and 4:should this not be change-add-conveyance-reference-number as there will also be a conveyance-reference-number  page